front/src/containers/Home/index.js

```diff
@@ -0,0 +1,38 @@
+//react imports
+import React from 'react'
+import { useNavigation } from '@react-navigation/native';
+
+//local imports
+import {
+  Container,
+  LogoView,
+  Logo,
+  ContainerBtn,
+  ButtonLogin,
+  ButtonRegister,
+  Text
+} from './styles'
+import logo from '../../assets/logo.png'
+
+const Home = () => {
+
+  const navigation = useNavigation();
+
+  return (
+    <Container>
+      <LogoView>
```

Comment:
You don't need around logo with a view, can set the style to center and margin in own logo component